pulumi-yaml.sln

```diff
@@ -0,0 +1,644 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
It's back :sweat_smile:  Might be worth adding this to `.gitignore`

Reply:
Yeh lets do that